packages/dx-react-grid/docs/reference/table-view.md

```diff
@@ -94,6 +96,18 @@ value | any | The value to be rendered within the cell
 row | [Row](grid.md#row) | Specifies a table row
 column | [Column](#column) | Specifies a table column
 
+### <a name="table-row-args"></a>TableRowArgs
+
+Describes properties passed to a row template when it is being rendered.
+
+A value with the following shape:
+
+Field | Type | Description
+------|------|------------
+tableRow | [TableRow](#table-row) | Specifies a table row
+children | ReactElement | A React element used to render a table row.
+style? | Object | Styles that should be applied to the root row element
```

Comment:
What about the row argument? I think it is much more intuitive to use a user-defined row instead of accessing it from the row field of the tableRow interface.